blocks/dashboard/renderSiteList.js

```diff
@@ -53,23 +53,40 @@ async function fetchProjects(token, type = 'googleDrive', scrollTo = false) {
     });
     const { projects, pagination } = await response.json();
     sitesList.innerHTML = `
-            ${title}
-            <ul class="my-sites-overview" data-totalitems=${pagination.totalItems}>
-              ${projects.map(({ projectSlug, projectName, projectDescription }) => `
-                <li>
-                  <a href="/${isDarkAlley ? 'da-site' : 'site'}/${projectSlug}/overview">
-                    <div class="project-thumbnail" data-src="https://${projectSlug}.${KESTREL_ONE}"></div>
-                    <div class="project-content">
-                      <h2 title="${projectName}">${projectName}</h2>
-                      <p title="${projectSlug}"><strong>${projectSlug}</strong></p>
-                      <p class="project-description" title="${projectDescription || ''}">${projectDescription || ''}</p>
-                    </div>
-                  </a>
-                </li>
-              `).join('')}
-            </ul>
-            ${paginator(pagination.totalItems, currentLimit, actualPage)}
-          `;
+    ${title}
+    <ul class="my-sites-overview" data-totalitems=${pagination.totalItems}></ul>
+    ${paginator(pagination.totalItems, currentLimit, actualPage)}
+    `;
+
+    const ul = sitesList.querySelector('.my-sites-overview');
+    sitesList.append(ul);
```

Comment:
I don't understand why here you're appending the `ul` in `sitesList` ? Isn't it already a child of it ? Also why can't you use `safeText` instead of `textContent/innerText` and `value`?